src/spaces/openstreetmap_new.jl

```diff
@@ -358,6 +358,127 @@ end
 plan_route!(agent::A, dest::Int, model; kwargs...) where {A<:AbstractAgent} =
     plan_route!(agent, (dest, dest, 0.0), model; kwargs...)
 
+"""
+    function distance(pos_1, pos_2, model::ABM{<:OpenStreetMapSpace})
```

Comment:
you shouldn't include `function` in the call declaration. I think a better name is `osmdistance`. Distance is missleading as it typically means euclidean distance.

Reply:
If we're keeping the `OSM` module, doesn't `OSM.distance` already accomplish this?

Speaking of, all the module docstrings should be `OSM.(...)` to be in line with other submodules.

Reply:
Oh yeah, then yes, if the module OSM is kept, which I agree with, all docstrings must start with `OSM.`...